package.json

```diff
@@ -8,7 +8,6 @@
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "dev": "webpack-dev-server --open --mode=development",
-    "precommit": "lint-staged",
```

Comment:
Let's bring the linting for commits back in so it'll help everyone refactor changes based on eslint/prettier. If there are eslint errors preventing you from moving past this point, some rules may be disabled with comments or the eslintrc.json may also be modified if it's a rule entirely you guys would not like to follow